soc/elements/vexriscv/nitrogen/Kconfig.soc

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2025 aesc silicon
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_NITROGEN
+	bool
+	select SOC_FAMILY_ELEMENTS_VEXRISCV
+
+config SOC_ELEMRV_N
+	bool
+	select SOC_SERIES_NITROGEN
+
+config SOC_SERIES
+	default "nitrogen" if SOC_SERIES_NITROGEN
+
+config SOC
+	default "elemrv-n" if SOC_ELEMRV_N
```

Comment:
underscores not dashes in soc names

Reply:
Is the target name meant to be `board/soc/cpu`? I'm wondering if I should do `elemrv/elemrv/n` instead of `elemrv/elemrv_n`

Reply:
> Is the target name meant to be `board/soc/cpu`?

See [Board terminology](https://docs.zephyrproject.org/latest/hardware/porting/board_porting.html#board-terminology)

> I'm wondering if I should do `elemrv/elemrv/n` instead of `elemrv/elemrv_n`

`elemrv/elemrv/n` if the SOC can have different CPU configurations, and most of the other SOC related configurations are the same (i.e. possibility to have 32/64 bit CPUs, or different extensions)